resources/css/filament/app/theme.css

```
@@ -635,6 +640,255 @@
+        /* Zera só o padding: a régua inferior de 1px do fi-contained é
+           exatamente o trilho que o layout pede. */
+        padding: 0;
+        overflow-x: auto;
+        overflow-y: hidden;
+        scrollbar-width: none;
+        -ms-overflow-style: none;
+
+        &::-webkit-scrollbar {
+            display: none;
+        }
+        background-color: transparent;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)